phpunit/block-supports/layout-test.php

```diff
@@ -501,7 +501,47 @@ public function data_layout_support_flag_renders_classnames_on_wrapper() {
 						),
 					),
 				),
-				'expected_output' => '<p class="wp-container-content-1">Some text.</p>', // The generated classname number assumes `wp_unique_prefixed_id( 'wp-container-content-' )` will not have run previously in this test.
+				'expected_output' => '/<p class="wp-container-content-[a-f0-9]{8}">Some text.<\\/p>/',
```

Comment:
md5 is predictable so it should stay the same between the test runs. What is the reasoning behind using pattern matching here? I expected the tests would ensure that these values don't accidentally change as long as params provided remain the same.

Reply:
No problem. I can change that. I wasn't sure if it would be a good idea to update the tests every time the passed parameters change, but that's not going to happen often, right? 😅

PS: I copied what is being done for the [duotone classes](https://github.com/WordPress/gutenberg/blob/7a90b32cfef27873e2ed069776d9f05d8e37785b/phpunit/class-wp-duotone-test.php#L45) (although, in that case, the IDs might not be predictable).

Reply:
I've just updated the tests to check the generated hashes with their expected values.

Reply:
Cool, it seems like it's doable for the use cases in this PR. We could leave a note in the tests that the hash might change if the conditions provided get refined in the future.